.github/workflows/appsignals-e2e-eks-test.yml

```diff
@@ -158,15 +165,6 @@ jobs:
               ${{ inputs.aws-region }} \
               ${{ env.SAMPLE_APP_NAMESPACE }}
           
-              # If the workflow provides a specific ADOT image to test, patch the deployment and restart CW agent related pods
-              if [ ${{ inputs.appsignals-adot-image-name }} != "" ]; then
-                kubectl patch deploy -namazon-cloudwatch amazon-cloudwatch-observability-controller-manager --type='json' \
-                -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/args/0", "value": "--auto-instrumentation-java-image=${{ inputs.appsignals-adot-image-name }}"}]'
-          
-                kubectl delete pods --all -n amazon-cloudwatch
-                kubectl wait --for=condition=Ready pod --all -n amazon-cloudwatch
-              fi
-          
```

Comment:
Why remove?

Reply:
This part of the code was for release testing on ADOT. It won't be used in this repository anymore

Reply:
Doesn't ADOT release testing refer to this repo now though?

Reply:
NVM, discussed offline and the change makes sense. ADOT repo will have its own workflow that will have this section as needed.